content/ember/v3/component-is-visible.md

```diff
@@ -5,60 +5,124 @@ until: '4.0.0'
 since: '3.15'
 ---
 
-In ember earlier versions, we had the concept of 'A controller should never call method or change property on its associated view '( Ember.View class)', instead the view should bind the state of its associated controller.'
+Classic components have a number of APIs to handle the wrapper `div` that they create by default.
+One of them is `isVisible`, which controls if the component is hidden to the end user or not.
+`isVisible` is now deprecated in accordance with [RFC #324](https://github.com/emberjs/rfcs/blob/master/text/0324-deprecate-component-isvisible.md).
 
-So, we were using something like this,
+You can update your component in one of two ways, you can wrap your component's template in an `{{if}}`, or you can use the [`hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) HTML attribute.
```

Comment:
```suggestion
You can update your component in one of two ways, you can wrap your component's template in an `{{if}}`, or you can use the [`hidden`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) HTML attribute.

It's worth noting that not all visibility approaches are equal- we recommend reviewing use of `aria-hidden` as well as accessible ways to visibly hide content while still making it available to assistive technology. 
```

Reply:
Is there some `aria-hidden` documentation we can link to? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute says it's draft status.

Reply:
https://www.w3.org/WAI/PF/aria/states_and_properties#aria-hidden is what it looks like,  but that link does say editor's draft. I'd have to look for the one that isn't the draft but for all practical purposes the content there is legit.